### PR TITLE
Add initial sync for Rutter, which occurs when the integration is first run

### PR DESCRIPTION
**Pull request by ScopeyNZ**
Requires a new unreleased version of `@chec/integration-handler`

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head><link rel="stylesheet" href="/index.f2f6e04f.css">
+  <meta charset="UTF-8">
+  <title>Configuration App</title>
```
[the remark this answers is not in the document]
Comment (ScopeyNZ) [reply dist/index.html]: Na but I can change it for the curious.

**Review by robbieaverill — APPROVED** [review]
My comments around pagination for the products API call to Chec disappeared, but make sure you consider that at some point

**Comment by ScopeyNZ** [comment]
I didn't see any comments, but we're only fetching products from the Rutter API right now, so we only have to worry about their pagination. It's a good point though, there is pagination so I'll fix that.

**Comment by robbieaverill** [comment]
Yeah weird, maybe I didn't save them

**Comment by ScopeyNZ** [comment]
Added support for Rutter's cursor pagination.

**Review by robbieaverill — APPROVED** [review]
LGTM. In case my other comments were lost, from memory I suggested that when you add `category_count` to the response you should do it before the list of `product_ids`, since the response body is truncated when you view it in the API. It feels like the counts are more important than the IDs when checking to see if it worked.

**Pull request merged**